bot/exts/help_channels/_channel.py

```diff
@@ -26,19 +26,20 @@ def get_category_channels(category: discord.CategoryChannel) -> t.Iterable[disco
 
 
 async def get_closing_time(channel: discord.TextChannel) -> datetime:
-    """Return the timestamp at which the given help `channel` should be closed."""
+    """Return the timestamp at which the given help channel, `channel`, should be closed."""
     log.trace(f"Getting the closing time for #{channel} ({channel.id}).")
 
-    if await _message.is_empty(channel):
+    if is_empty := await _message.is_empty(channel):
```

Comment:
Does this need a walrus operator? Why not just make a variable in front?

Reply:
I doesn't *need* one, but is this not what the walrus operator is for?

Reply:
it may be that i just generally don't like the walrus operator. i have never seen it make the code more readable.